admin/conf/application.conf

```diff
@@ -22,8 +22,17 @@ akka {
       }
     }
   }
+
+  blocking-operations {
+    executor = "thread-pool-executor"
+    throughput = 10
+    thread-pool-executor {
+      fixed-pool-size = 128
+    }
+  }
```

Comment:
What's the reasoning behind using a fixed size thread-pool-executor dispatcher?

Reply:
It's ideal for blocking operations